src/components/berths/Berth/SelectedBerth/index.tsx

```diff
@@ -77,7 +76,7 @@ class SelectedBerth extends Component<Props, State> {
               })}
             >
               <span key={berth.identifier}>
-                {index + 1}. {getLocalizedText(berth.name, intl.locale)}
+                {index + 1}. {berth.name}
```

Comment:
Are you sure about berth name shouldn't be translated here? I bet it doesnt need to, just asking